nni/tools/nnictl/command_utils.py

```diff
@@ -34,8 +34,7 @@ def check_output_command(file_path, head=None, tail=None):
 def kill_command(pid):
     """kill command"""
     if sys.platform == 'win32':
-        process = psutil.Process(pid=pid)
-        process.send_signal(signal.CTRL_BREAK_EVENT)
+        psutil.Process(pid).terminate()
```

Comment:
For windows platform, main.ts monitor `SIGBREAK` signal to trigger stop process, does this terminate() function in win32 trigger `SIGBREAK`?